crates/lumina_server/src/lobby/in_game.rs

```diff
@@ -57,16 +59,22 @@ fn init_spaceship_position(
     q_in_game_lobbies: Query<(), With<LobbyInGame>>,
     terrain_config: TerrainConfig,
     lobby_infos: Res<LobbyInfos>,
+    mut team_toggle: ResMut<TeamToggle>, // Access the team toggle resource
 ) {
     let Some(terrain_config) = terrain_config.get() else {
+        eprintln!("Terrain config is not available!");
         return;
     };
 
-    // TODO: Use different positions for different ships based on their team id.
-    let width = terrain_config.tile_size * terrain_config.noise_surr_width as f32;
-    let desired_position = Vec2::splat(width);
+    let (bottom_left, upper_right) =
+        TerrainStates::get_map_corners_without_noise_surr(terrain_config);
+
+    println!(
+        "\n\n\n\nDebugging Map Corners: Bottom-left: {:?}, Upper-right: {:?}",
+        bottom_left, upper_right
+    );
 
-    for (mut position, id, entity) in q_spaceships.iter_mut() {
+    for (mut position, _rotation, id, entity) in q_spaceships.iter_mut() {
```

Comment:
```suggestion
    for (mut position, team_type, id, entity) in q_spaceships.iter_mut() {
```

If rotation not in use, dun use it.